seeding/index.js

```diff
@@ -0,0 +1,34 @@
+import { startDBConnection } from '../server/config';
+import logger from '../server/config/winston';
+import { seedUsers, seedVendors } from './user.seeding';
+
+const seedDB = async () => {
+  const model = process.env.SEED;
```

Comment:
We should have a constant that defines all the models that we can seed in the database. For now, it would contain USER, RESET, UNKNOWN.
```suggestion
  const model = process.env.MODEL || MODEL.UNKNOWN;
```